incubator/sapho/templates/deployment.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+spec:
+  replicas: {{ .Values.replicaCount }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+    spec:
+      containers:
+      - name: {{ template "fullname" . }}
+        image: "{{ .Values.image }}"
+        imagePullPolicy: {{ default "" .Values.imagePullPolicy | quote }}
+        env:
+        - name: SAPHO_DB_HOSTNAME
+          value: {{ template "mysql.fullname" . }}
+        - name: SAPHO_DB_PORT
+          value: {{ .Values.saphoDBport | quote }}
+        - name: SAPHO_DB_TYPE
+          value: {{ .Values.saphoDBtype }}
+        - name: SAPHO_DB_USERNAME
+          value: {{ .Values.saphoDBuser }}
+        - name: SAPHO_DB_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: {{ template "mysql.fullname" . }}
+              key: mysql-root-password
+        ports:
+        - name: http
+          containerPort: 8080
+#        livenessProbe:
```

Comment:
Yes, I think you are right. I will test and push the changes.Thanks for all the help, sorry I am kind of a noob haha...